manim/config/config.py

```diff
@@ -150,3 +152,14 @@ def _parse_config(config_parser, args):
         _init_dirs(file_writer_config)
 config = _parse_config(config_parser, args)
 camera_config = config
+
+# Set the different loggers
+set_rich_logger(config_parser["logger"])
+if file_writer_config["log_to_file"]:
+    # IMPORTANT note about file name : The log file name will be the scene_name get from the args (contained in file_writer_config). So it can differ from the real name of the scene.
```

Comment:
Question: what if the user doesn't pass a scene name? `manim -pl scene.py`. I'd use the `input_file` to create the name.

Also typo: get -> gotten

Reply:
Yes, good idea, but the advantage of this is when there is several scenes in one file and the user wants to keep logs from each scenes, without having only one log file for the whole scenes file.

But I think this is a better idea. Should we use the file name instead of the scene name passed as arg ?

Reply:
I'm fairly neutral about this. In my experience, it is rare to have more than one Scene in the same file in real use cases. It also depends on the size of the log files.

OTOH, if we ever implement parallel rendering of scenes, we may want to keep the logs separated. So let's keep it like this :shrug:

Reply:
Done  :D

I also _hate_ this formatting.